keras/ops/numpy_test.py

```diff
@@ -9,15 +9,16 @@
 from keras.backend.common.keras_tensor import KerasTensor
 from keras.backend.common.variables import ALLOWED_DTYPES
 from keras.ops import numpy as knp
+from keras.testing.test_utils import named_product
 
 # TODO: remove reliance on this (or alternatively, turn it on by default).
 np_config.enable_numpy_behavior()
 
 
 class NumpyTwoInputOpsDynamicShapeTest(testing.TestCase):
     def test_add(self):
-        x = KerasTensor((None, 3))
-        y = KerasTensor((2, None))
+        x = KerasTensor([None, 3])
```

Comment:
Canonically, we use tuples for shapes (and only accept lists for user-friendliness). In general any non-mutable iterable should be a tuple rather than a list.

Reply:
Yes, in general we use tuples for shape, which I really like.

For whatever reason, in this file and for `KerasTensor(...)` specifically, it was 90% lists. It's slightly more readable because of the braces are different for the constructor and the shape.

But I'll flip them the other way (all tuples) for consistency.